packages/date-picker/test/keyboard-input.test.js

```diff
@@ -135,23 +128,6 @@ describe('keyboard', () => {
       expect(datepicker.invalid).to.be.false;
       expect(datepicker.checkValidity()).to.be.true;
     });
-
-    it('should empty value with invalid input', async () => {
```

Comment:
**note:** Moved to [`form-input.test.js`](https://github.com/vaadin/web-components/pull/4163/files#diff-746ffa5fcc10c6bab313c794447622d931d27c361bb3c354f0135efabf98046fR200)